website/docs/index.html.markdown

```diff
@@ -26,6 +26,8 @@ The following Cloud Director versions are supported by this provider:
 * 10.3
 * 10.4
 
+Also Cloud Director Service (CDS) is supported.
```

Comment:
Do we need to specify backing VCD version here or not really useful?

Reply:
After more discussion, this statement seems great and simple for users to understand (i.e. that we support same CDS' VCD versions as VCD on-prem).